Network/Network.h

```diff
@@ -142,6 +178,21 @@ namespace WPEFramework {
             uint16_t m_stunCacheTimeout;
             bool m_stunSync;
             uint32_t m_apiVersionNumber;
+            bool m_useIpv4WifiCache;
+            bool m_useIpv6WifiCache;
```

Comment:
can you use these bool variables as ATOMIC as these can be updated by the IARM Event in different thread context.